client/src/components/TermsOfAgreementModal.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react';
+import {Button, Modal } from 'semantic-ui-react';
+
+/*
+    props:
+    - modalOpen
+    - closeModal: ()
+*/
+export default class TermsOfAgreementModal extends Component {
+    constructor(props){
+        super(props)
+    }
+
+    render() {
+        return (
+            <Modal
+                open={this.props.modalOpen}
+                style={{overflow: 'auto', maxHeight: 600}}
+            >
+                // TODO: Replace with actual terms
```

Comment:
Move this TODO comment to before the `render() { ...` as this is showing up in the modal
![image](https://user-images.githubusercontent.com/31665569/89360673-31ac4280-d697-11ea-9b27-f30d30bc485c.png)